freqtrade/freqai/data_kitchen.py

```diff
@@ -450,6 +454,31 @@ def split_timerange(
         # print(tr_training_list, tr_backtesting_list)
         return tr_training_list_timerange, tr_backtesting_list_timerange
 
+    def split_timerange_live_models(
+        self
+    ) -> Tuple[list, list]:
+
+        tr_backtesting_list_timerange = []
+        pair = self.pair.split("/")[0].split(":")[0]
```

Comment:
i'm not sure what you're trying to do here - but it's for sure wrong.

a pair is either `BTC/USDT` - or `BTC/USDT:USDT`.

the first expression will (in both instances) give you `BTC` - so the 2nd is irrelevant.
It's however also not a pair, but just the base asset ... so i'm not if this is really giving the desired outcome (as well as if it's even necessary).
if the expectation is BTC - then the 2nd split is unnecessary.

The variable name is also missleading (maybe `asset` would be better) - as this is no longer a pair.

Reply:
You are right. I need only the asset. I will fix the code and var name.